src/lib/wind_estimator/python/wind_est_derivation.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov  1 19:14:39 2016
+
+@author: roman
+"""
+
+from sympy import *
+
```

Comment:
@jgoppert I'll see what I can do. I never used ipython notebook before but I know it's cool.